charts/div-vs/values.aat.template.yaml

```diff
@@ -0,0 +1,9 @@
+java:
+    environment:
+        REFORM_ENVIRONMENT: "aat"
+        APPLICATION_INSIGHTS_IKEY: "da42f3e5-892a-4201-a2df-05d2c28a410b"
+        SLOT: "PRODUCTION"
+        REFORM_SERVICE_NAME: "vs"
+        REFORM_TEAM: "div"
+        APPINSIGHTS_INSTRUMENTATIONKEY: "da42f3e5-892a-4201-a2df-05d2c28a410b"
```

Comment:
I don't think we need `REFORM_SERVICE_NAME` and `REFORM_TEAM` as they're in the default `values.template.yaml`

Reply:
Deleted and moved  _SLOT_ to _values.template.yaml_

Reply:
Not sure if SLOT will always be production though

Reply:
Might be okay for now..

Reply:
TBH I'm not really sure if it's used on AKS

Reply:
Removed